pkg/indexers/apibinding_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package indexers
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/kcp-dev/logicalcluster/v3"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apisv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+)
+
+func TestIndexAPIBindingByAPIExport(t *testing.T) {
+	tests := map[string]struct {
+		obj     interface{}
+		want    []string
+		wantErr bool
+	}{
+		"not an APIBinding": {
+			obj:     "not an APIBinding",
+			want:    []string{},
+			wantErr: true,
+		},
+		"has a workspace reference": {
+			obj: &apisv1alpha1.APIBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						logicalcluster.AnnotationKey: "root:default",
+					},
+					Name: "foo",
+				},
+				Spec: apisv1alpha1.APIBindingSpec{
+					Reference: apisv1alpha1.BindingReference{
+						Export: &apisv1alpha1.ExportBindingReference{
+							Path: "root:workspace1",
```

Comment:
Would add a test case where Path is unset here

Reply:
done